Kentor.AuthServices.StubIdp/Models/MetadataModel.cs

```diff
@@ -10,7 +10,7 @@ namespace Kentor.AuthServices.StubIdp.Models
 {
     public static class MetadataModel
     {
-        public static ExtendedEntityDescriptor CreateIdpMetadata(bool includeCacheDuration = true)
+        public static ExtendedEntityDescriptor CreateIdpMetadata(bool defaultPost = false, bool includeCacheDuration = true)
```

Comment:
Two bool params with default values look a bit error prone, especially when you add the new one first. Please remove the defaults (for both of them)